include/tvm/script/ir_builder/tir/ir.h

```diff
@@ -411,8 +411,10 @@ Var EnvThread(String thread_tag, DataType dtype = DataType::Int(32));
  * \param buffer The buffer.
  * \param value The value to be stored.
  * \param indices The indices location to be stored.
+ * \param predicate A vector mask of int1 values indicating which lanes of a vector are to be
```

Comment:
Can we specify that the number of lanes for the `predicate` must be equal to the number of lanes in the `value`?